content/docs/integrations/istio/ambient-ingress.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Ingress to ambient mesh
+weight: 30
+description: Use kgateway as the ingress gateway for your ambient mesh. 
+---
+
+Use kgateway as the ingress gateway for your ambient mesh. 
+
+## About ambient mesh
+
+Solo collaborated with Google to develop ambient mesh, a new “sidecarless” architecture for the Istio service mesh. Ambient mesh uses node-level ztunnels to route and secure Layer 4 traffic between pods with mutual TLS (mTLS). Waypoint proxies enforce Layer 7 traffic policies whenever needed. To onboard apps into the ambient mesh, you simply label the namespace the app belongs to. Because no sidecars need to be injected in to your apps, ambient mesh significantly reduces the complexity of adopting a service mesh. 
+
+To learn more about ambient, see the [ambient mesh documentation](https://ambientmesh.io/docs/about/). 
+
+## About this guide
+
+In this guide, you learn how to use kgateway as the ingress gateway to route traffic to the httpbin app that is part of an ambient service mesh. 
+
+This guide assumes that you run your ambient mesh in a single cluster and want to use kgateway as the ingress gateway to protect your ambient mesh services. 
+
+{{< reuse-image src="img/ambient-ingress.svg" width="600px" caption="kgateway as an ingress gateway to your ambient mesh" >}}
+
```

Comment:
Are there any limitations we should note here? Like using kgateway as a waypoint is not yet supported, but you can use the standard ambientmesh.io waypoints instead?

Reply:
not sure yet, that is next on my list :D